source/manual/github-unavailable.html.md

```diff
@@ -6,125 +6,117 @@ layout: manual_layout
+### Making changes to code in AWS CodeCommit before deployment
 
-1. Push your new tag to CodeCommit by running `git push aws release_XYZ`
+If you need to deploy from AWS CodeCommit because GitHub is down or you need to test a change of a sensitive nature, you will likely need to push a new branch of code to CodeCommit.
 
-### Authenticating with Jenkins
+Log into the Tools account (you may need to contact a GOV.UK AWS admin user to be granted access):
 
-If GitHub.com is down, we will not be able to log in to Jenkins.
+```
+gds aws govuk-tools-admin -l
+```
 
-In this scenario, Jenkins security should be disabled to enable deployment:
+Switch to the `eu-west-2` region, and navigate to the [CodeCommit Repositories screen](https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories).
 
-1. SSH to the Jenkins deploy instance:
+Find the relevant repo and click the "HTTPS" Clone URL to copy the clone URL, then clone the repository locally. For example:
 
-```console
-gds govuk connect -e production ssh aws/jenkins
+```
+git clone https://git-codecommit.eu-west-2.amazonaws.com/v1/repos/account-api
 ```
 
-2. Disable Puppet: `govuk_puppet -r "Emergency Jenkins deploy" --disable`
-3. Edit the Jenkins configuration file: `sudo vim /var/lib/jenkins/config.xml`
-4. Replace `<useSecurity>true</useSecurity>` with `<useSecurity>false</useSecurity>` and save
-5. Restart Jenkins: `sudo service jenkins restart`
+Make the necessary changes in a branch locally. For example:
 
-Note that once security is disabled, anyone on GDS trusted IPs will be able to deploy to that environment. This will bypass protection for Production - do not leave Production without security for any longer than necessary.
+```
+git checkout -b hotfix
+rm README.md
+git add .
+git commit -m "Test commit: remove README"
+git push -u origin HEAD
+```
 
-See the [Jenkins documentation](https://jenkins.io/doc/book/system-administration/security/#disabling-security) for further details.
+After you've pushed the branch, you should see it in AWS CodeCommit.
 
-### Deploying the code change
+### Deploying a hotfix from AWS CodeCommit
 
-1. Review the pull request on AWS CodeCommit through the [AWS Console](https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories?region=eu-west-2) (access to GOV.UK repos must be granted by a GDS AWS administrator).
+Once you've [pushed your hotfix branch to CodeCommit](#making-changes-to-code-in-aws-codecommit-before-deployment), you should:
 
-1. Create a release tag manually in git. This should follow the standard format
-   `release_X`. Tag the branch directly instead of merging it.
+1. Create a pull request in the CodeCommit UI.
+1. Ask another developer to review it as a sense check.
+1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
```

Comment:
In response to https://github.com/alphagov/govuk-developer-docs/pull/3738/files#r1005348807:

```suggestion
1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
  - In the example above, you can check the README was removed by SSH'ing into the relevant machine and executing `ls /var/apps/{REPO_NAME}` to confirm that the README is absent.
```

Reply:
👍